code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -9,9 +9,6 @@
 	var/obj/item/handcuffed = null //Whether or not the mob is handcuffed
 	var/obj/item/legcuffed = null  //Same as handcuffs but for legs. Bear traps use this.
 
-	var/list/features = list("mcolor" = "FFF", "tail" = "Smooth", "snout" = "Round", "horns" = "None", "frills" = "None", "spines" = "None", "body_markings" = "None")
-
-

```

Comment:
There is a separate features var for /datum/preferences, this is the one that changes when you modify you features in the lobby window. By the time preferenecs/copy_to(human) is called the human mob has already been created. So there's no need for a carbon/var/features, I've replcaed all instances of it by dna.features w/o issue.